src/main/webapp/dashboard.js

```diff
@@ -236,7 +241,7 @@ function createMenteeCardImage(imgSrc) {
  * Call functions to populate page sections with data.
  */
 function setUpPage() {
-  getUser(dummyGitHubID).then(createAboutMe);
+  getUser().then(createAboutMe);
```

Comment:
You need to add a catch to this fetch request to handle the errors coming from the fetch request. I'd suggest doing this before the `.then(createAboutMe)`